### PR TITLE
New url dfd

### PR DESCRIPTION
**Pull request by 01Parzival10**
depends on #https://github.com/DataFlowAnalysis/DFD-Metamodel/pull/8

### CODE REVIEW

File: .project
Comment (Entenwilli) [inline .project:1]: This file should not be included (this causes Eclipse to recognize an additional, non-existing, project)

File: tests/.project
Comment (Entenwilli) [inline tests/.project:1]: This file should not be included (this causes Eclipse to recognize an additional, non-existing, project)

**Review by Entenwilli — CHANGES_REQUESTED** [review]
Some project files snuck their way in :wink:

**Comment by Nicolas-Boltz** [comment]
@01Parzival10 Tests are failing. Seems that test models need to be fixed.

**Comment by Entenwilli** [comment]
There was an `"` missing in the online shop 😉

**Review by Entenwilli — APPROVED** [review]
Now it should work :D

**Pull request merged**